### PR TITLE
FE AX info level

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] In Hold - customer verification

## Related Issues
fixes: https://github.com/demisto/etc/issues/34563

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/fireeye/Integrations/integration-fireeye.yml [hunk 1]
```diff
@@ -260,6 +262,7 @@ script:
            httpParams.Password = params.credentials.password;
         }
         full_url = server + url + (method === 'GET' && method ? encodeToURLQuery(args) : '')
+        console.log(full_url);
```
Comment (ShahafBenYakir) [inline Packs/fireeye/Integrations/integration-fireeye.yml:265]:
```suggestion
        logDebug('Using the following url: ' + full_url);
```

**Comment by ShahafBenYakir** [comment]
Can we add this to the test PB?

**Comment by bakatzir** [comment]
@ShahafBenYakir No instance, so no.
the solution dod not worked out so i'll do it on a call with a customer.

**Comment by bakatzir** [comment]
Closing, re opening at: https://github.com/demisto/content/pull/11876

**Pull request closed without merging**